storm-core/src/clj/org/apache/storm/daemon/common.clj

```diff
@@ -86,26 +40,6 @@
 (defn new-executor-stats []
   (ExecutorStats. 0 0 0 0 0))
```

Comment:
Yes, I recommend to just use the thrift classes directly when we work on relative modules. It can save much effort now for the converting between clojure and jave. Please refer to my comments at the beginning of this PR.